README.rst

```diff
@@ -30,7 +30,8 @@ This is easily achieved by downloading
 Usage Example
 =============
 
-see examples in githup repository: https://github.com/adafruit/Adafruit_CircuitPython_STMPE610/exmaples
+See examples in githup repository: https://github.com/adafruit/Adafruit_CircuitPython_STMPE610/tree/master/examples
```

Comment:
Can you fix the other typo to say github? Thanks

Reply:
It is not clear to me why this change is necessary or correct? What is wrong with the original URL?  (Other than the githup in the comment)

Reply:
@makermelissa -  Yes, I can.  Funny that I missed that.
@jerryneedell  - The URL lead to a 404 error page.

Reply:
@jerryneedell, https://github.com/adafruit/Adafruit_CircuitPython_STMPE610/exmaples went to a 404 not found page.

Reply:
Yes- never mind- I see now that the update is correct —and it works!  Sorry for the confusion.